assignment_3/Marksheet/src/Marksheet.java

```diff
@@ -0,0 +1,148 @@
+import java.util.Scanner;
+
+
+public class Marksheet
+{
+	/**
+	 * 
+	 * @param grades of students
+	 * @return Average of grades
+	 */
+	public float Calculate_Average(int grades[])
+	{
+		float Average=0;
```

Comment:
average instead of Average.
follow camel casing for all the variable and method names